builtins/xrootd/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ endif()
 ExternalProject_Add(
     XROOTD
     URL ${XROOTD_SRC_URI}
-    URL_HASH SHA256=e01d83997e428580aadefd4b18cc6246e97a1941dfceff3726ffd0c40921de34
+    URL_HASH SHA256=3556d5afcae20ed9a12c89229d515492f6c6f94f829a3d537f5880fcd2fa77e4
```

Comment:
```console
$ curl -sLO http://xrootd.org/download/v5.5.1/xrootd-5.5.1.tar.gz | sha256sum 
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  -
```

so once https://sft.its.cern.ch/jira/browse/SPI-2231 is in then these should match for what will be the target at https://lcgpackages.web.cern.ch/tarFiles/sources/xrootd-5.5.1.tar.gz.

Reply:
Done: https://sft.its.cern.ch/jira/browse/SPI-2231?focusedCommentId=115619&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-115619